### PR TITLE
Add page describing how to report vulnerabilites

### PR DESCRIPTION
**Pull request by stigtsp**

### CODE REVIEW

**Review by Leont — APPROVED** [review]

File: docs/report.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+- Confirm the problem and assess severity
+- Check the code to find any potential similar problems.
+- Prepare fixes and coordinate a release
+
+Please allow for some time for authors and potential downstream distributions to
+coordinate fixes before going public. It is not unusual for
+authors to request extentions to any proposed disclosure date.
+
+
```
Comment (sjn) [inline docs/report.md:81]: #### Step 2b: When the Author is unresponsive

File: docs/report.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+### Step 3: Disclosure
+
+When disclosure has been agreed by the parties, or if details of the
+vulnerability has otherwise been made public, the CPAN Security WG will take
+following actions:
+
+1. Add a public record to CPAN vulnerability databases
+2. Register a CVE number, if relevant
+
```
Comment (sjn) [inline docs/report.md:100]: Should we also commit to post something on blogs.perl.org or other channels?

File: docs/report.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+- social media or chat channels, like IRC or Mastodon
+- mailing lists or discussion forums
+
+
+### Step 1: Prepare a Report
+
+Please provide a detailed description of the steps required to reproduce the
+vulnerability.
+
+The following information is useful:
```
Comment (sjn) [inline docs/report.md:51]: I think "Distribution name, version and package URL" is *required*, and not just "useful"...

**Review by sjn — APPROVED** [review]
Happy to accept this if my two comments are addressed.

**Comment by stigtsp** [comment]
> Happy to accept this if my two comments are addressed.

Feel free to add them as code suggestions :)

**Comment by stigtsp** [comment]
Merging this for now, @sjn feel free to patch the text :)

**Pull request merged**